components/feature/awesomebar/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="switch_to_tab_description">Switch to tab</string>
+</resources>
```

Comment:
Please make sure this file is listed in `l10n.toml` so that our translation process will pick it up.